polyfill/lib/casts.mjs

```diff
@@ -0,0 +1,140 @@
+import { ES } from './ecmascript.mjs';
+
+export function absolute(arg) {
```

Comment:
any reason not to have these casting functions be `static from` on each class? like `Absolute.from(arg)`

Reply:
I really like them, because they make a functional style much nicer. (Experimental for now)

Reply:
If they're not static, how would they work with subclassing?

Reply:
I like the idea of static from methods. At the same time, I hadn't really thought through interaction with subclassing. I guess it would new this with the positional arguments?

Reply:
@littledan itd presumably use the same mechanism as Promise.resolve and Array.from